Sources/Experiences/UI/ScreenViewLayout.swift

```diff
@@ -45,7 +47,8 @@ class ScreenViewLayout: UICollectionViewLayout {
         needsPreparation = false
     }
     
-    open func prepare(frame: CGRect) {
+    // swiftlint:disable:next cyclomatic_complexity function_body_length
+    func prepare(frame: CGRect) {
```

Comment:
@seanrucker should this actually be open?

Reply:
Probably not. I can’t think of a reason to override it. You’d essentially be creating a completely new layout if you did that.

Reply:
👍 boom